src/Controls/src/Core/Compatibility/Handlers/ListView/Windows/CellControl.cs

```diff
@@ -320,7 +312,14 @@ void OpenContextMenu()
 				FlyoutBase.SetAttachedFlyout(CellContent, flyout);
 			}
 
-			FlyoutBase.ShowAttachedFlyout(CellContent);
+			FlyoutBase
+				.GetAttachedFlyout(CellContent)
+				.ShowAt(
+					CellContent,
+					new FlyoutShowOptions
+					{
+						Position = point,
```

Comment:
This is the key part of the change: to actually show the menu where the user clicked/tapped. All the rest is plumbing changes.